charts/fluentd/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: fluentd
 description: A Helm chart for Kubernetes
 # type: application
-version: 0.2.1
-appVersion: v1.11.5
+version: 0.2.2
+appVersion: v1.12.0
```

Comment:
The `v` prefix isn't part of a SemVer 2 version so shouldn't be included here.

```suggestion
appVersion: 1.12.0
```

Reply:
you merged anyway. Can you open an Issue so we dont forget to remove this?

Reply:
I merged?

Reply:
i did it